osu.Game/Rulesets/UI/Scrolling/ScrollingHitObjectContainer.cs

```diff
@@ -252,14 +252,14 @@ private void updateLayoutRecursive(DrawableHitObject hitObject)
                 updateLayoutRecursive(obj);
 
                 // Nested hitobjects don't need to scroll, but they do need accurate positions and start lifetime
-                updatePosition(obj, hitObject.HitObject.StartTime);
+                updatePosition(obj, hitObject.HitObject.StartTime, hitObject.HitObject.StartTime);
```

Comment:
This doesn't look like it'll handle hitobjects nested at multiple levels, and maybe a better way of handling this is something like:
```csharp
private void updateLayoutRecursive(DrawableHitObject hitObject, double? parentHitObjectStartTime = null)
{
    parentHitObjectStartTime ??= hitObject.HitObject.StartTime;

    ...

    foreach (var obj in hitObject.NestedHitObjects)
    {
        updateLayoutRecursive(obj, parentHitObjectStartTime);
        updatePosition(obj, hitObject.HitObject.StartTime, parentHitObjectStartTime);
    }
}

Reply:
I thought we basically weren't supporting multi-level nesting as it is not used anywhere?

Reply:
I could be misremembering, but I don't see why it shouldn't be supported? This class already supports it, this change breaks it, that's all.